Sources/NIO/BaseSocketChannel.swift

```diff
@@ -276,9 +279,9 @@ class BaseSocketChannel<T: BaseSocket>: SelectableChannel, ChannelCore {
 
     // This is `Channel` API so must be thread-safe.
     public final var pipeline: ChannelPipeline {
-        return self.lifecycleManager.channelPipeline
+        return self._pipeline
     }
-
+    
```

Comment:
Sneaky sneaky whitespace.

Reply:
whaat, I thought I have git configured to auto-remove them

Reply:
ha, that was on my other laptop. /me should sync dotfiles :)